pandas/io/formats/string.py

```diff
@@ -0,0 +1,200 @@
+"""
+Module for formatting output data in console (to string).
+"""
```

Comment:
The content of this module and class StringFormatter was extracted from DataFrameFormatter.
Without substantial refactor only the parts related to console/string formatting were moved out of DataFrameFormatter.